src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumDeserializer.java

```diff
@@ -0,0 +1,29 @@
+package org.sagebionetworks.bridge.sdk;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdScalarDeserializer;
+
+@SuppressWarnings("serial")
+class LowercaseEnumDeserializer extends StdScalarDeserializer<Enum<?>> {
+
+    protected LowercaseEnumDeserializer(Class<Enum<?>> clazz) {
+        super(clazz);
+    }
+
+    @Override
+    public Enum<?> deserialize(JsonParser jp, DeserializationContext ctxt) throws IOException, JsonProcessingException {
+        String text = jp.getText().toUpperCase();
+        try {
+            Method valueOfMethod = handledType().getDeclaredMethod("valueOf", String.class);
+            return (Enum<?>) valueOfMethod.invoke(null, text);
+        } catch (Exception e) {
+            throw new RuntimeException("Cannot deserialize enum " + handledType().getName() + " from " + text, e);
+        }
```

Comment:
Consider using Enum.valueOf(clazz, text) http://docs.oracle.com/javase/7/docs/api/java/lang/Enum.html#valueOf%28java.lang.Class,%20java.lang.String%29 instead of Reflection.

Also, instead of catching all Exceptions, you should catch only the exceptions you expect to come up. (In the case of Enum.valueOf(), it's IllegalArgumentExceptions and NullPointerExceptions. If you stick to Reflection, there's also ReflectiveOperationException.)

Also, wrapping in RuntimeException means your tests can't distinguish between this exception and something generic like NullPointerException. Consider using an existing RuntimeException that meets your usecase (like IllegalArgumentException or JsonProcessingException).